vendor/modules.txt

```diff
@@ -307,6 +307,9 @@ github.com/hashicorp/go-cty-funcs/crypto
 github.com/hashicorp/go-cty-funcs/encoding
 github.com/hashicorp/go-cty-funcs/filesystem
 github.com/hashicorp/go-cty-funcs/uuid
+# github.com/hashicorp/go-getter v1.4.1
```

Comment:
Packer uses go-getter@v2. I added a comment to the file where this was introduced